### ISSUE TITLE
[Bug]: Spinner component fails accessibility. Need to add Stroke Alpha 2, Modify Brand Stroke 2 high contrast values.

### ISSUE THREAD

**Issue by redongreen** [microsoft/fluentui#28385]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-spinner--default
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-spinner--default

### Context 
Currently [spinner component](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-spinner--default) primary variant is not accessible in high contrast.

![CleanShot 2023-06-29 at 11 48 05](https://github.com/microsoft/fluentui/assets/20573265/6d8b73d4-b1e4-4f7e-b3c1-13be2b2a3314)

This is partially due to the fact that we never created a token to support the look and feel of our spinner spec. The spec originally instructed engineers to hard code a value in place.

Full HC audit can be found here; [Figma link](https://www.figma.com/file/hJmjsqTApez3dLzTr2Q17U/Fluent-2-web-(V4-test)?type=design&node-id=95789%3A234196&mode=design&t=tbfXhEHcaAbByIDu-1)

### To do

To fix the spinner, we need to add two new stroke tokens

#### Add new token

Add new token `Neutral Stroke Alpha 2` 
Light theme: White 20% alpha
Dark theme: White 20% alpha
High contrast theme: hcCanvas #000000

![CleanShot 2023-06-29 at 11 44 14@2x](https://github.com/microsoft/fluentui/assets/20573265/7ba34eaa-a857-43c0-859a-11e2df4d8645)

Add new token `Brand Stroke 2 Contrast` 
Light theme: Comm 140 #B4D6FA
Dark theme: Comm 50 #0E4775
High contrast theme: hcCanvas #000000

![CleanShot 2023-07-10 at 16 30 58@2x](https://github.com/microsoft/fluentui/assets/20573265/feffcdbd-99b9-425b-a559-b8afaba389b1)

#### Update existing component
Ensure that we are following the updated token assignment. [Link to component spec.](https://www.figma.com/file/6kXOvdH8mTflgW2lGFYaEq/Spinner?type=design&node-id=2043%3A1044&mode=design&t=TRAGiC6lFOKWhdFE-1)

![CleanShot 2023-07-10 at 16 36 15@2x](https://github.com/microsoft/fluentui/assets/20573265/40203611-1513-458a-b5d7-218cb2aed389)

cc @daisygeng

### Logs

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-spinner--default
```

### Requested priority

High

### Products/sites affected

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-spinner--default

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by behowell** [comment]
@miroslavstastny can you help @tomi-msft with adding the new token? Thanks!

**Comment by redongreen** [comment]
I updated the ticket based on @miroslavstastny audit findings.

### Usage conflict
Brand Stroke 2 is used as a track in spinner component; this requires a black color in high contrast.
Ex.

![CleanShot 2023-07-10 at 16 25 47@2x](https://github.com/microsoft/fluentui/assets/20573265/5adc8e63-bcae-4422-a945-9ed1f81ed711)


Brand Stroke 2 is also used as a stroke outline for the badge; this requires a white color in high contrast. 
Ex.

![CleanShot 2023-07-10 at 16 26 31@2x](https://github.com/microsoft/fluentui/assets/20573265/12ed76f7-42ed-4613-8446-bce603bb7ac3)

Therefore, we will need a new token to provide brand tint color in light and dark theme while optimizing for the current high-contrast theme. 

I propose that we add a new token `Brand Stroke 2  Contrast`

![CleanShot 2023-07-10 at 16 28 05@2x](https://github.com/microsoft/fluentui/assets/20573265/3bad562e-7c63-4db8-98dd-91f44e1239a3)
